ChatClient.go

```diff
@@ -97,9 +119,12 @@ func (user *ChatClient) Join(name string) {
 
 // List : Lists all available channels
 func (user *ChatClient) List() {
+	// Maybe use a more descriptive variable name than 'what' ;)
```

Comment:
haha old habit from Node - when passing a value to log/send/write/etc and not sure what it is, I always use `what`